source/extensions/tracers/zipkin/span_buffer.h

```diff
@@ -16,15 +22,24 @@ class SpanBuffer {
   /**
    * Constructor that creates an empty buffer. Space needs to be allocated by invoking
    * the method allocateBuffer(size).
+   *
+   * @param version The selected Zipkin collector version.
```

Comment:
Would be nice to have a `{@see}` maybe to the link where the versions are declared. :surfing_man: